src/stringBuffer.js

```diff
@@ -24,7 +24,15 @@
  * @return {function}
  */
 function makeBuffer() {
-  // write code here
+  let bufferStr = '';
+
+  return (someText = undefined) => {
```

Comment:
you do not need default undefined, it is to replace undefined with some other value